zero/dg_calc_gk_rad_vars.c

```diff
@@ -126,23 +128,25 @@ void gkyl_dg_calc_gk_rad_vars_nI_nu_advance(const struct gkyl_dg_calc_gk_rad_var
 
     long loc_conf = gkyl_range_idx(conf_range, idx);
     long loc_phase = gkyl_range_idx(phase_range, idx);
-
-    gkyl_rect_grid_cell_center(&up->phase_grid, idx, xc);
-
-    const double* vnu_surf_d = gkyl_array_cfetch(vnu_surf, loc_phase);
-    const double* vnu_d = gkyl_array_cfetch(vnu, loc_phase);
-    const double* vsqnu_surf_d = gkyl_array_cfetch(vsqnu_surf, loc_phase);  
-    const double* vsqnu_d = gkyl_array_cfetch(vsqnu, loc_phase);   
-
-    const double *nI_d = gkyl_array_cfetch(nI, loc_conf);
-
-    double* nvnu_surf_d = gkyl_array_fetch(nvnu_surf, loc_phase);
-    double* nvnu_d = gkyl_array_fetch(nvnu, loc_phase);
-    double* nvsqnu_surf_d = gkyl_array_fetch(nvsqnu_surf, loc_phase);  
-    double* nvsqnu_d = gkyl_array_fetch(nvsqnu, loc_phase);   
-
-    up->rad_nI_nu(vnu_surf_d, vnu_d, vsqnu_surf_d, vsqnu_d, nI_d, 
-      nvnu_surf_d, nvnu_d, nvsqnu_surf_d, nvsqnu_d);
+    const double* vtsq_d = gkyl_array_cfetch(vtsq, loc_conf);
+    if ( vtsq_d[0] * up->cellav_norm_conf > vtsq_min ) {
```

Comment:
an alternative is to build this norm factor into the vtsq_min array, so that you don't have to multiply this every time step.